src/components/Board.tsx

```diff
@@ -0,0 +1,29 @@
+import { Grid, Paper } from "@mui/material";
+import React from "react";
+
+const arr = Array.from({ length: 90 }, (_, i) => i + 1);
+
+const numberToBeDisplayed: (num: number) => string = (num: number) =>
+  num < 10 ? `0${num}` : `${num}`;
+
+const Board: React.FC = () => {
+  return (
+    <Grid
```

Comment:
wrap the grid component around paper

Reply:
because if item is not of type paper there is distortion in ui like this

![image](https://user-images.githubusercontent.com/44979100/139525118-5dae811f-5ff9-4d67-8870-6fa3432acf29.png)
with paper it works like this

Reply:
you cant add padding between grid items cells